internal/exec/exec.go

```diff
@@ -303,6 +303,9 @@ func (r *request) addError(err *errors.QueryError) {
 func (r *request) handlePanic() {
 	if err := recover(); err != nil {
 		execErr := errors.Errorf("graphql: panic occured: %v", err)
+		if err, ok := err.(error); ok {
+			execErr.ResolverError = err
```

Comment:
My thought here is that this could help your use case of not leaking internals. The error message could say something like "internal error" but more information (not available to client) could be included on the ResolverError (maybe thats a bad name) which won't get sent to client because it wont be serialized to json.

Also, if something panics in a resolver, won't it eventually be caught by this func? I don't have a use case for this yet but I could see myself wanting to have access to the originating error? 

I don't feel very strongly about this so if you'd like me to remove it then I will.